contrib/build-push-ceph-container-imgs.sh

```diff
@@ -201,7 +201,7 @@ function create_registry_manifest {
   # This should normally work, by the time we get here the arm64 image should have been built and pushed
   # IIRC docker manisfest will fail if the image does not exist
   for image in daemon-base daemon; do
-    for ceph_release in luminous mimic; do
+    for ceph_release in luminous mimic nautilus; do
```

Comment:
Maybe we can use https://github.com/ceph/ceph-container/pull/1381/files#diff-9bd28db8c71e92c690fefd2d189c6b46R15 and remove the first element of the list?

Reply:
This will only work if CEPH_RELEASES is configured with the default value. https://github.com/ceph/ceph-container/blob/master/contrib/build-push-ceph-container-imgs.sh#L13-L16

If someone set CEPH_RELEASES with only one value then this solution won't work (but I don't know if it's a valid scenario)

Reply:
It's not intended to change `CEPH_RELEASES` so I think we are good.

Reply:
ack